strings/is_substring_palindrome.py

```diff
@@ -0,0 +1,61 @@
+MAX_LEN = 1000
+
+text = ""
+dp = [[False for i in range(MAX_LEN)] for i in range(MAX_LEN)]
+
+def preprocess(s):
+    """

```

Comment:
Very strange format of the docstring. Also it's better to add a description what the function exactly does